strawberryfields/api/connection.py

```diff
@@ -278,3 +278,5 @@ def __repr__(self):
 
     def __str__(self):
         return self.__repr__()
+
+connection = Connection()
```

Comment:
What is the reason for this change here? If this file is being imported when `strawberryfields` is imported, than it seems that a connection object will attempt to be created on import. This will cause error messages for the user if the configuration does not exist

Reply:
That's a fair point! This was added such that pinging could be used in the CLI. Previously no such object was created. Maybe it would be better to create a `Connection` object only when pinging is used with the CLI?

Reply:
Sounds good!